sdk/monitor/Azure.Monitor.Ingestion/samples/Sample1_HelloWorld.md

```diff
@@ -0,0 +1,15 @@
+# <scenario_list>
+
+To use these samples, you'll first need to set up resources. See [getting started](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.Ingestion/README.md#getting-started) for details.
+
+## <scenario>
+
+You can create a client and call the client's `<operation>` method
+
+```C# Snippet:Azure_Monitor_Ingestion_Scenario
```

Comment:
You should wrap a snippet around the simplest "Hello World" of your tests to dump it here so anyone who wants to play with your prototype can see how.  It's okay if it's not the final sample we end up with when you write the README.  See https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets for how to do that if it's been a bit.